Inferences/Superposition.cpp

```diff
@@ -178,7 +179,7 @@ bool Superposition::checkClauseColorCompatibility(Clause* eqClause, Clause* rwCl
   }
   if(getOptions().showBlocked()) {
     env.beginOutput();
-    env.out()<<"Blocked superposition of "<<eqClause->toString()<<" into "<<rwClause->toString()<<endl;
+    env.out()<<"Blocked superposition of "<<eqClause->toString()<<" into "<<rwClause->toString()<<std::endl;
```

Comment:
This is close to being just picky, but why the inconsistency here?

Reply:
Superposition.cpp uses std in a bit clunky way.
It imports it in
```
#if VDEBUG
#include <iostream>
using namespace std;
#endif
```
But if VDEBUG is not turned on it fails.
I didn't want to have a file which has 2 `using namespace std;`
So I added `using std::pair;`
But then it still uses one `cout` outside of VDEBUG cc (which I think is a mistake) so for a moment I just added std:: to this cout.

Reply:
I wouldn't mind turning the conditional using into unconditional one. (And the printing uder showBlocked seems fine even in release). But this is all really just cosmetics.